sdk/include/asm/asm.inc

```diff
@@ -172,6 +172,20 @@ ENDM
     .skip size, fill
 ENDM
 
+lgdt32A MACRO address
+    DB 00Fh
+    DB 001h
+    DB 015h
+    DD address
+ENDM
+
+lidt32A MACRO address
+    DB 00Fh
+    DB 001h
+    DB 01Dh
+    DD address
+ENDM
+
```

Comment:
```suggestion
#define lgdt lgdt fword ptr
#define lidt lidt fword ptr
```

Reply:
i removed this but i added the lXdtPrefix. see the updated code.